features/support/command_helpers.rb

```diff
@@ -16,7 +16,7 @@ def assertions
     WARNINGS = {
       "FC001" => "Use strings in preference to symbols to access node attributes",
       "FC002" => "Avoid string interpolation where not required",
-      "FC003" => "Check whether you are running with chef server before using server-specific features",
+      # FC003 was yanked and is considered reserved, do not reuse it
```

Comment:
This is just copy pasta from the last FC rule we deprecated. The actual docs for FC word is differently. This is just a code comment to prevent devs from reusing the number